src/settings_internal.js

```diff
@@ -30,7 +30,13 @@ var SIDE_MODULE_EXPORTS = [];
 var SIDE_MODULE_IMPORTS = [];
 
 // Like EXPORTED_FUNCTIONS, but will not error if symbol is missing
-var EXPORT_IF_DEFINED = ['__start_em_asm', '__stop_em_asm'];
+// The start/stop symbols are included by default for the benefit of the
+// PostEmscripten pass in binaryen that strips the EM_ASM and EM_JS strings
+// from the binary.  They are included here rather than in REQUIRED_EXPORTS
+// because not all programs contains EM_JS or EM_ASM data section, in which
+// case these symbols won't exist.
```

Comment:
Please add a comment that these exports will be removed by PostEmscripten.

Reply:
Done. I also updated the comment to reflect the fact that we actually need to these in the python metadata extraction process.